inst/rmd/qc_report.rmd

```diff
@@ -157,25 +161,65 @@ unfiltered_celldata <- data.frame(colData(unfiltered_sce)) %>%
+  geom_point(aes(size = filter_status), alpha = 0.2) +
   scale_x_log10(labels = scales::label_number()) +
   scale_y_log10(labels = scales::label_number()) + 
   scale_color_manual(values = c("darkgreen", "grey80")) + 
-  scale_shape_manual(values = c(16, 1)) + # filled and unfilled circles
+  scale_size_manual(values = c(2, 1)) +
   labs(
     x = "Rank", 
     y = "Total UMI count",
     color = "Filter status",
-    shape = "Filter status"
+    size  = "Filter status"
   ) + 
   guides(color = guide_legend(override.aes = list(alpha = 1))) + 
-  theme(plot.margin = margin(rep(20, 4)),
-        legend.position = c(0, 0),
+  theme(legend.position = c(0, 0),
+        legend.justification = c(0,0),
+        legend.background = element_rect(color = "grey20", size = 0.25),
+        legend.box.margin = margin(rep(5, 4)))
+```
+
+```{r, fig.alt="Smoothed knee plot of filtered and unfiltered droplets"}
+grouped_celldata <- unfiltered_celldata %>%
+  select(sum, rank, filter_status) %>%
+  mutate(rank_group = floor(rank / 100) )%>%
+  group_by(rank_group) %>%
+  summarize(
+    med_sum = median(sum),
+    med_rank = median(rank),
+    pct_passed = sum(filter_status == "Passed") / n() * 100
+  )
+
+top_celldata <- unfiltered_celldata %>%
+  select(sum, rank, filter_status) %>%
+  filter(rank <= 50) %>%
+  mutate(filter_status = ifelse(filter_status == "Passed", 100, 0))
+
+ggplot(grouped_celldata, aes(x = med_rank, y = med_sum, color = pct_passed)) + 
+  geom_point(mapping = aes(x = rank, y = sum, color = filter_status),
+             data = top_celldata,
+             alpha = 0.5) + 
+  geom_line(size = 2, lineend = "round", linejoin= "round") +
+  scale_x_log10(labels = scales::label_number()) +
+  scale_y_log10(labels = scales::label_number(accuracy = 1)) + 
+  scale_color_gradient2(low = "grey70", 
+                        mid = "forestgreen", 
+                        high = "darkgreen", 
+                        midpoint = 50) +
+  labs(
+    x = "Rank", 
+    y = "Total UMI count",
+    color = "% passing\ncell filter"
+  ) + 
+  theme(legend.position = c(0, 0),
         legend.justification = c(0,0),
         legend.background = element_rect(color = "grey20", size = 0.25),
         legend.box.margin = margin(rep(5, 4)))
 ```
 
+The total UMI count of each droplet (barcode) plotted against the rank of that droplet allows visualization of the distribution of sequencing depth across droplets. 
+The droplets that are expected to contain cells were identified with [`DropletUtils::emptyDrops()`](https://bioconductor.org/packages/release/bioc/html/DropletUtils.html) , which uses both the total UMI counts and expressed gene content ([Lun  _et al._ 2019](https://doi.org/10.1186/s13059-019-1662-y)).
+As the boundary between droplets passing and failing this filter does not correspond only to the axes of this plot, some regions contain droplets in both categories, so we here color the plot by the percentage passing the filter.
```

Comment:
I might restate the fact that the expressed gene content is used for filtering in the second sentence here. Repetition is good.

Reply:
When I read this (in the rendered form, where I can't see code), I do find myself asking how we get the `rank_group` exactly but I'm not our user.

Reply:
Sorry I said rank group as a matter of convenience (I can see the code in the diff after all). I basically meant how do we get the averages.